### PR TITLE
chore: add comments

### PR DESCRIPTION
**Pull request by hamza-ghufran**
Define Kallos

### CODE REVIEW

**Comment by hamza-ghufran** [comment]
`This is a PR comment`

**Review by hamza-ghufran — COMMENTED** [review]
`This is a Pr review`

**Comment by hamza-ghufran** [comment]
> `This is a Pr review`

Would this be a review comment or pr comment ?

EDIT: This is A Pr Comment

File: index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
-function kallos(){}
+function kallos(){
+  console.log('Greek word for beauty')
+}
```
Comment (hamza-ghufran) [inline index.js:3]: This is a review comment.

EDIT: This was added as a review. with state = `COMMENTED` and body = ''

Comment (hamza-ghufran) [reply index.js:3]: Is this also added as a review or review comment ?

EDIT: added as a review comment

**Review by calli-human — CHANGES_REQUESTED** [review]
This is a review with: `Request Changes`

**Comment by calli-human** [comment]
> This is a review with: `Request Changes`

This is a pr comment from requester.

File: index.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 
 function kakos(){}
```
Comment (calli-human) [inline index.js:5]: This is review from other. [add a single comment]

Comment (calli-human) [reply index.js:5]: Startng a review

**Review by calli-human — CHANGES_REQUESTED** [review]

**Review by calli-human — APPROVED** [review]
This is review

**Pull request closed without merging**